lib/Pages/AddItem.dart

```diff
@@ -1,10 +1,248 @@
+                    border: OutlineInputBorder(),
+                    hintText: 'Describe what you want to list',
+                  ),
+                  validator: (String? value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter some text';
+                    }
+                    return null;
+                  },
+                ),
+                const Text(
+                  "Deal Method",
+                  textAlign: TextAlign.left,
+                ),
+                Wrap(
+                  spacing: 5.0,
+                  children: ["Meet up", "Delivery"].map((String tag) {
+                    return FilterChip(
+                      label: Text(tag),
+                      selected: _dealOptionSelected.contains(tag),
+                      onSelected: (bool value) {
+                        setState(() {
+                          if (value) {
+                            if (!_dealOptionSelected.contains(tag)) {
+                              _dealOptionSelected.add(tag);
+                            }
+                          } else {
+                            _dealOptionSelected.removeWhere((String name) {
+                              return name == tag;
+                            });
+                          }
+                        });
+                      },
+                    );
+                  }).toList(),
+                ),
+                const Text(
+                  "Location",
+                  textAlign: TextAlign.left,
+                ),
+                TextFormField(
+                  controller: locationController,
+                  decoration: const InputDecoration(
+                    border: OutlineInputBorder(),
+                    hintText: 'Where do you want to meet up?',
+                  ),
+                ),
+                ElevatedButton(
+                    onPressed: () {
+                      if (_formKey.currentState!.validate()) {
+                        var db = FirebaseFirestore.instance;
+                        var listing = Provider.of<ListingProvider>(context,
+                            listen: false);
+                        final item = {
+                          "category": listing.category,
+                          "condition": listing.condition,
+                          "secondCategory": _secondCategorySelected,
+                          "dealOption": _dealOptionSelected,
+                          "description": descriptionController.text,
+                          "images":
+                              listing.items.map((e) => e.downloadUrl).toList(),
+                          "title": titleController.text,
+                          "credits": int.tryParse(creditsController.text) ?? 0,
+                          "size": _clothesSize,
+                          "location": locationController.text,
+                          // "owner": FirebaseAuth.instance.currentUser!.uid,
```

Comment:
Could this field be added back?

Reply:
Also would it be possible to add "target" for Male, Female, Unisex? As well as init the status as "OPEN"